src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -83,6 +83,8 @@ def delete(self, pool_name):
     @pool_task('edit', ['{pool_name}'])
     def set(self, pool_name, flags=None, application_metadata=None, configuration=None, **kwargs):
         self._set_pool_values(pool_name, application_metadata, flags, True, kwargs)
+        if kwargs.get('pool'):
+            pool_name = kwargs['pool']
```

Comment:
Why not using explicit optional args vs. hidden interfaces (`**kwargs`), and meaningful variable names (`pool` vs. `new_pool_name`)?:
```diff
-  def set(self, pool_name, flags=None, application_metadata=None, configuration=None, **kwargs):
+  def set(self, pool_name, flags=None, application_metadata=None, configuration=None, new_pool_name=None, **kwargs):
    self._set_pool_values(pool_name, application_metadata, flags, True, kwargs)
-    if kwargs.get('pool'):
-      pool_name = kwargs['pool']
+   if new_pool_name:
+      pool_name = new_pool_name
```

Reply:
This would need a change in frontend and would break `_set_pool_values` where the pool actually is renamed.